apps/agent/src/main/workers/push-activities.ts

```
@@ -294,6 +303,29 @@ class PushActivities {
+			for (let i = 0; i < notSyncTimer.length; i++) {
+				const timerOffline = notSyncTimer[i].timer;
+				if (timerOffline?.isStartedOffline) {
+					await this.apiService.startTimer({
+						organizationId: authConfig?.user?.employee?.organizationId,
+						startedAt: timerOffline.startedAt,
+						tenantId: authConfig?.user?.employee?.tenantId,
+						organizationTeamId: null,
+						organizationContactId: null
+					});
+					await this.timerService.remove({ id: timerOffline?.id });
+					continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)